src/libs/OptionsListUtils.ts

```diff
@@ -1827,7 +1827,7 @@ function getOptions(
         return option;
     });
 
-    const havingLoginPersonalDetails = options.personalDetails.filter((detail) => !!detail?.login && !!detail.accountID && !detail?.isOptimisticPersonalDetail);
+    const havingLoginPersonalDetails = includeP2P ? options.personalDetails.filter((detail) => !!detail?.login && !!detail.accountID && !detail?.isOptimisticPersonalDetail) : [];
```

Comment:
based on your comment shouldn't we cleanup `includePersonalDetails` if it's redundant?

Reply:
@getusha I updated to cleanup `includePersonalDetails`.

Reply:
Nice I like that. 

NAB: We could remove the variable `havingLoginPersonalDetails` since its unnecessary.